pfdp/css/cta.css

```diff
@@ -0,0 +1,33 @@
+.cta-text {
+  font-weight: 600;
+  font-size: 32px;
+  line-height: 39px;
+  text-align: center;
+}
+
+.btn-primary {
```

Comment:
name of the btn class should be btn-bm, check what's already existing in the global css file and adapt to your need